lib/js/dom.nim

```diff
@@ -238,6 +246,13 @@ type
     lastModified*: int
     name*: cstring
 
+  # https://developer.mozilla.org/en-US/docs/Web/API/HTMLTextAreaElement
```

Comment:
nit: might as well put this link as a doc comment instead of comment? ditto elsewhere

Reply:
I don't know why it had these comments there, it seems obvious where to go get the docs. I am just trying to be consistent. 

I would like to remove all comments of that type?

Reply:
having them in docs removes friction

Reply:
I think it just adds extra noise to parse the definitions - packed with unless URLs.

Reply:
if you're gonna delete all those links at the very least add 1 link in the existing module level doc comment on top:

```nim
## Declaration of the Document Object Model for the `JavaScript backend
## <backends.html#backends-the-javascript-target>`_.
## See  also https://developer.mozilla.org/en-US/docs/Web/API
```
(added 3rd line)

Reply:
I have made all comments pointing to documentation be ##doc comments are requested.